docs/develop/support-ie-11.md

```diff
@@ -39,50 +43,76 @@ There are two popular transpilers. Both of them can work with source files that
 
 Importing the entire core-js library will import all core-js features. You can also import only the polyfills that your Office Add-in requires. For instructions about how to do this, see [CommonJS APIs](https://github.com/zloirock/core-js#commonjs-api). The core-js library has most of the polyfills that you need. There are a few exceptions detailed in the [Missing Polyfills](https://github.com/zloirock/core-js#missing-polyfills) section of the core-js documentation. For example, it doesn't support `fetch`, but you can use the [fetch](https://github.com/github/fetch) polyfill.
 
 For a sample add-in that uses core.js, see [Word Add-in Angular2 StyleChecker](https://github.com/OfficeDev/Word-Add-in-Angular2-StyleChecker).
 
-## Determine at runtime if the add-in is running in Internet Explorer
+## Test an add-in on Internet Explorer
 
-Your add-in can discover if it is running in Internet Explorer by reading the [window.navigator.userAgent](https://developer.mozilla.org/docs/Web/API/Navigator/userAgent) property. This enables the add-in to either provide an alternate experience or gracefully fail. The following is an example. Note that Internet Explorer sends a string beginning with "Trident" as the value of userAgent.
+See [Internet Explorer 11 testing](../testing/ie-11-testing.md).
 
-```javascript
-if (navigator.userAgent.indexOf("Trident") === -1) {
+# [Microsoft Edge Legacy](#tab/edge)
 
-    // IE is not the browser. Provide a full-featured version of the add-in here.
+## Troubleshoot Microsoft Edge Legacy issues
 
-} else {
+If you encounter issues as you develop your add-in to support Microsoft Edge Legacy, see the "Troubleshoot Microsoft Edge issues" section of [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md#troubleshoot-microsoft-edge-issues) for guidance.
+
+## Debug an add-in that supports Microsoft Edge Legacy
+
+To debug your add-in that supports Microsoft Edge Legacy, use one of the following options.
+
+- [Debug add-ins using developer tools in Microsoft Edge Legacy](../testing/debug-add-ins-using-devtools-edge-legacy.md)
+- [Debug add-ins using the Microsoft Office Add-in Debugger Extension for Visual Studio Code](../testing/debug-with-vs-extension.md)
 
-    // IE is the browser. So here, do one of the following: 
-    //  1. Provide an alternate experience that does not use any of the HTML5
-    //     features that are not supported in IE.
-    //  2. Enable the add-in to gracefully fail by putting a message in the UI that
-    //     says something similar to: 
-    //      "This add-in won't run in your version of Office. Please upgrade 
-    //      either to perpetual Office 2021 or to a Microsoft 365 account."          
+---
+
+## Determine at runtime if the add-in is running in IE11 or Microsoft Edge Legacy
```

Comment:
Should this section go earlier in the article?

Reply:
That makes sense. I'll move it up.